data/pug.txt

```diff
@@ -150,6 +151,7 @@ outfit "Tier 3 Anti-Missile"
 		"reload" 6
 		"firing energy" 80
 		"firing heat" 10
+	description "Although they serve different roles, this powerful anti-missile turret seems to have a close resemblance to the Pug Gridfire Turret."
```

Comment:
I don't understand this description. The sprites are very different-looking and they serve different purposes (anti-ship vs anti-missile)

Reply:
How about now? "Despite they are very different, this powerful anti-missile turret seems to function in a way similar to the Pug Gridfire Turret."

Reply:
Changed again, now it is: "Despite being very different in appearance, this powerful anti-missile turret seems to function in a way similar to the Pug Gridfire Turret."

Reply:
I guess I don't understand why any mention of the Gridfire turret needs to be made at all. It's not obvious how they might function in a similar way, nor does this information provide the player with any useful information or function as flavor text.

Reply:
but that's what flavor texts are for, no? we don't really care about the ongoing "war" between lovelace's AM- and sidewinder-team, yet [there it is](https://github.com/endless-sky/endless-sky/blob/master/data/weapons.txt#L730).

Reply:
I'm saying that this isn't actually flavorful enough to count as flavor text. :) It's just an unexplained technical detail that's sort of confusing and context-less.

Reply:
Alright, removed. How is it now? "This powerful anti-missile turret is capable of destroying even the strongest of the missiles."

Reply:
*`the strongest of missiles`, unless my english fails me here